app/src/main/java/org/piwigo/accounts/UserManager.java

```diff
@@ -256,4 +257,10 @@ public void updateAccount(@NonNull Account account, @NonNull String url, String
             }
         }
     }
+
+    public void removeAccount(){
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP_MR1) {
```

Comment:
what if we are on a smaller SDK?

Reply:
@ramack I saw other comments and my question is. Should i add support here for lower versions  or support for them will be cutted ?

Reply:
Adding a version check is good, but we should document in a comment why we need it and what we do on older versions. 'Nothing can be a valid choice if the missing functionality is not essential. In Some such situations we might want to notify the user. In this specific case as we anyhow rely on SDK >= 21 I guess we can just skip it..'

Reply:
ok